rosco-web/src/main/groovy/com/netflix/spinnaker/rosco/controllers/BakeryController.groovy

```diff
@@ -176,10 +174,11 @@ class BakeryController {
         def existingBakeStatus = queryExistingBakes(bakeKey)
 
         if (existingBakeStatus) {
-          Id bakesId = registry.createId('bakes').withTag("duplicate", "true")
-          registry.counter(bakesId).increment()
+          registry.counter(registry.createId("bakesRequested", ["flavor":"duplicate"])).increment()
 
           return existingBakeStatus
+        } else {
+          registry.counter(registry.createId("bakesRequested", ["flavor":"plain"]))
```

Comment:
'plain' sounds a little weird. Maybe 'new' or something like that? I don't feel that strongly about it, but it isn't very descriptive.